test/.eslintrc.json

```diff
@@ -1,4 +1,6 @@
 {
+	"extends": "../src/.eslintrc.json",
+	"root": true,
```

Comment:
Yeah, I guess duplicating of globals would suck more.

>  What's the drawback of what I proposed?

Angular historically didn't shy away from those in their repo so it's seems fine for them, but for me personally that seems as a clutter